09-redux/classworks/Malkov_Aleksei/src/pages/Routes.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Switch, Route, Redirect } from 'react-router-dom';
+
+import useRedditApi from 'hooks/useRedditApi';
+
+import Login from 'pages/login';
+import Callback from 'pages/callback';
+import Home from 'pages/home';
+import Search from './search';
+
+function Routes({ reactSubreddit, about, search }) {
```

Comment:
now, when we use Redux we do not pass props to Routes and further/. we use Redux